instaclustr/structs.go

```diff
@@ -16,49 +16,57 @@ type Bundle struct {
 	Options BundleOptions `json:"options,omitempty" mapstructure:"options"`
 }
 
+type OmitEmptyBool struct {
+	value bool
+}
+
+//func (b *OmitEmptyBool) UnmarshalJSON(data []byte) error {
```

Comment:
Do we need this?

Reply:
If not, do we need the type above?

Reply:
No, found a different way to solve this problem